test/examples_test.go

```diff
@@ -181,6 +181,18 @@ func getExamplePaths(t *testing.T, dir string) []string {
 			return filepath.SkipDir
 		}
 		if info.IsDir() == false && filepath.Ext(info.Name()) == ".yaml" {
+			// Ignore test matching the regexp in the TEST_EXAMPLES_IGNORES
+			// environement variable.
```

Comment:
Is there a way to do this with the "-run" flag to go test instead? Or would that be too difficult since you would need to basically specify a negative regular expression.

Reply:
Tha'ts right, that would be quite difficult and probably error prone, I would need to list all tests, filter them out and feed them back to go_test_e2e

Reply:
Makes sense.

Reply:
Typo: environment